include/CConcurrentObjectCache.h

```diff
@@ -129,9 +129,9 @@ namespace impl
 
         inline bool outputAll(size_t& _inOutStorageSize, MutablePairType* _out) const
         {
-            auto lk = lock_read();
-            const bool r = BaseCache::outputAll(_inOutStorageSize, _out);
-            return r;
+            //auto lk = lock_read();
+            //const bool r = BaseCache::outputAll(_inOutStorageSize, _out);
+            return true;
```

Comment:
???

Reply:
There is an issue that some methods of that class are declared `const` but call `lock_read()`/`lock_write()` which isn't `const`

Reply:
So for now i just removed `const` qualifiers/removed the `const` versions of the methods

Reply:
ah, so instead make sure locking doesnt require mutability (use `mutable` keyword or something)

Reply:
`mutable` on the lock or?

Reply:
yea